common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore_bulk_operations.py

```diff
@@ -15,6 +15,11 @@
 
 class TestBulkWriteMixin(unittest.TestCase):
     shard = 2
+    VERSION_GUID_DICT = {
+        'SAMPLE_VERSION_GUID': 'deadbeef1234' * 2,
+        'SAMPLE_UNICODE_VERSION_GUID': u'deadbeef1234' * 2,
+        'BSON_OBJECTID': ObjectId()
```

Comment:
These key strings are repeated often enough in the class that I'd go ahead and declare them as constants also (to make typos less likely, etc.)